src/Utils/DirectForeignKeyMethodDescriptor.php

```diff
@@ -144,6 +170,25 @@ private function getFilters(ForeignKeyConstraint $fk) : string
         return $parametersCode;
     }
 
+    /**
+     * Check if the ForeignKey have an unique index
+     *
+     * @return bool
+     */
+    private function hasLocalUniqueIndex(): bool
+    {
+        foreach ($this->getForeignKey()->getLocalTable()->getIndexes() as $index) {
+            if (
+                $index->isUnique()
+                && count($index->getColumns()) === count($this->getForeignKey()->getLocalColumns())
```

Comment:
You must use `$index->getUnquotedColumns()` and `$this->getForeignKey()->getUnquotedLocalColumns()`.
Otherwise, it causes problems for Oracle and columns in lowercase.

Haaaaa... oracle...

Reply:
Done :)